docs/features/record-routing.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Record routing'
+sidebar_position: 4
+keywords: [ 'synchronization', 'collections', 'routing' ]
+---
+
+## Overview
+
+Record routing in Conduit makes it easier to build one-to-many and many-to-many
+pipeline topologies. Or, in other words, it's possible to:
+
+* route data from one source to multiple collections in one destination
+* route data from multiple sources to multiple destinations.
+
+Through this, a number of use cases are possible:
+
+* Keep two instances synchronized. For examples, two database instances with
+  multiple tables in each can be kept in sync with a single pipeline with only
+  one source and one destination connector.
+* Route data from one collection (table, topic, index, etc.) to multiple
+  collections on a single server.
+* Route data to different instances or different systems altogether.
+
+This can be done it two ways:
```

Comment:
```suggestion
This can be done in two ways:
```

Reply:
Done in [51cba2e](https://github.com/ConduitIO/conduit-site/pull/79/commits/51cba2e59fe619c036c820a3d91b7e7f4b44fedd).